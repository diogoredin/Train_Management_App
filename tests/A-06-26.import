SERVICE|390|12.2|06:05|A1|06:13|A211|06:14|B4|06:26|A411|06:48|A511|06:57|A611|07:01|A711|07:06|A811|07:10|A9
SERVICE|690|12.2|07:06|B1|07:17|B2|07:31|B4|07:53|Z41|08:14|B5|08:16|B6|12:18|B7
SERVICE|180|12.2|08:06|Z1|08:17|Z2|08:31|Z5|08:53|Z41|09:14|Z51|09:24|Z6|09:38|B7
PASSENGER|Azura
PASSENGER|Boethiah
PASSENGER|Clavicus Vile
PASSENGER|Hermaeus Mora
PASSENGER|Hircine
PASSENGER|Malacath
PASSENGER|Mehrunes Dagon
PASSENGER|Mephala
PASSENGER|Meridia
