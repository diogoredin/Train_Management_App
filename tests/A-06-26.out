Menu principal
1 - Reiniciar
2 - Abrir
3 - Guardar
4 - Consulta de serviços
5 - Gestão de passageiros
6 - Gestão de itinerários
0 - Sair
Escolha uma opção: Menu de gestão de itinerários
1 - Mostrar todos os itinerários
2 - Mostrar itinerários associados a um passageiro
3 - Registar itinerário para um passageiro
0 - Sair
Escolha uma opção: Identificador do passageiro: Nome da estação de partida: Nome da estação de destino: Data de partida (YYYY-MM-DD): Hora de partida (HH:MM): 
Itinerário 1 para 2017-12-31 @ 12.91
Serviço #390 @ 1.69
06:05 A1
06:13 A211
06:14 B4
Serviço #690 @ 11.22
07:31 B4
07:53 Z41
08:14 B5
08:16 B6
12:18 B7
Número do itinerário: Menu de gestão de itinerários
1 - Mostrar todos os itinerários
2 - Mostrar itinerários associados a um passageiro
3 - Registar itinerário para um passageiro
0 - Sair
Escolha uma opção: Menu principal
1 - Reiniciar
2 - Abrir
3 - Guardar
4 - Consulta de serviços
5 - Gestão de passageiros
6 - Gestão de itinerários
0 - Sair
Escolha uma opção: 